src/scrabble-long.js

```diff
@@ -0,0 +1,134 @@
+class Scrabble {
+  constructor (word) {
+    this.word = word
+    this.points = 0
+  }
+
+  // first find out if the string is empty string, whitepsace or null, then score 0.
+  score () {
+    if (
+      this.word === '' ||
+      this.word === ' ' ||
+      this.word === ' \t\n' ||
+      this.word === null
+    ) {
+      this.points += 0
+      return 0
```

Comment:
do you need the `this.points += 0` here? i would've thought you could just use `return 0'